test/library/packages/Python/examples/torch/myModel.skipif

```diff
@@ -4,7 +4,7 @@
 if [ -n "$CHPL_TEST_VGRND_EXE" ] && [ "$CHPL_TEST_VGRND_EXE" == "on" ]; then
   echo "True"
   exit 0
-fi
+else
```

Comment:
It looks like you're still changing this file, but now in the other direction?  I'd try doing a `cherry-pick -n` of your changes onto a fresh branch from main and see if that gets the file change to not be committed (and still run fine in testing)

Reply:
It should be fixed now